.github/workflows/tests-clustermesh-upgrade.yaml

```diff
@@ -187,7 +190,9 @@ jobs:
             --set=clustermesh.apiserver.readinessProbe.periodSeconds=1 \
             --set=clustermesh.apiserver.kvstoremesh.readinessProbe.periodSeconds=1 \
             --set=clustermesh.apiserver.updateStrategy.rollingUpdate.maxSurge=1 `# Use surge update strategy to enable clients to failover` \
-            --set=clustermesh.apiserver.updateStrategy.rollingUpdate.maxUnavailable=0"
+            --set=clustermesh.apiserver.updateStrategy.rollingUpdate.maxUnavailable=0 \
+            --set=clustermesh.apiserver.tls.authMode=${{ matrix.cm-auth-mode }} \
```

Comment:
Ah, I personally prefer having the closing doublequote on a separate line, so that all the other lines are consistent with \ at the end. I feel that it helps avoiding to miss it (or leaving a leftover quote) when adding new flags. But happy to change if you have a strong preference.

Reply:
Gotcha! Makes sense.